### PR TITLE
Update vale

### PR DESCRIPTION
**Pull request by m4sterbunny**

### CODE REVIEW

File: spelling/README.md [hunk 1]
```diff
@@ -1,59 +1,81 @@
...
-If API specs are managed as YAML files, need to test whether Vale can handle linting YAMLs.
+Finally, there are more nuanced Consensys-specific styles such as substitutions, acronym overrides, and other configurations in the [Consensys Style folder](./styles/Consensys).
+
+The [vale.ini](vale.ini) file provides various switches to turn styles on and off and to set what file types are formatted. Furthermore, as part of the GHA, the downstream repos that use this can specify which folders Vale may lint.
 
 ## ToDo
 
+## Vale and yaml
+
+Vale may need a parser to handle linting YAMLs (e.g., PyYAML, ruyaml). Then Update the .vale.ini file to include the YAML style guide and specify the file extensions to lint.
```
Comment (m4sterbunny) [inline spelling/README.md:56]: @joshuafernandes I tested setting ini file to lint yaml, but that fails, think Vale would need a way to parse yamls first.

File: spelling/styles/Consensys/Punctuation.yml [hunk 2]
```diff
@@ -2,8 +2,12 @@ extends: substitution
...
 level: error
 ignorecase: true
+# This is not working
 action:
   name: replace
 swap:
-  ′: "'"
-  ″: '"'
\ No newline at end of file
+  '′': "'"
```
Comment (m4sterbunny) [inline spelling/styles/Consensys/Punctuation.yml:9]: @joshuafernandes I have tried various approaches to handle this a per @alexandratran request. Cannie do it. Can't find anything to suggest that Vale should be able to do this --> should be something the code linter does rather?

Comment (joshuafernandes) [reply spelling/styles/Consensys/Punctuation.yml:9]: I'll add this to my list and come back to this one @m4sterbunny

File: spelling/README.md (hunk 1 above)
Comment (joshuafernandes) [reply spelling/README.md:56]: I like this @m4sterbunny we can track these items here and work on them :)

**Review by joshuafernandes — APPROVED** [review]
LGTM

**Pull request merged**